blockchain/chain/src/store/chain_store.rs

```diff
@@ -306,6 +306,14 @@ where
         })?)
     }
 
+    pub fn unmark_block_as_validated(&self, cid: &Cid) -> Result<(), Error> {
+        let mut file = self.file_backed_validated_blocks.lock();
+        Ok(file.with_inner(|inner| {
+            // TODO(aatifsyed): should we return an error if block doesn't exist?
```

Comment:
I don't think it's an error if a previously validated block isn't in the set. The set is merely a cache and may be emptied at any point in time.